pull_request_template.md

```diff
@@ -1,7 +1,7 @@
-# Домашнее задание №
+# Домашнее задание №4
```

Comment:
https://github.com/track-mail-ru/homework#6-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D0%B5%D0%BC-%D1%88%D0%B0%D0%B1%D0%BB%D0%BE%D0%BD-%D0%B4%D0%BB%D1%8F-pr
![Uploading image.png…]()